### PR TITLE
Tests: Check different log driver can work with podman logs

### PR DESCRIPTION
**Pull request by ypu**
Add a check step in podman run --log-driver test. Prefer to add it here as it already has a loop to cover all different drivers
in this test.

### CODE REVIEW

File: test/system/030-run.bats [hunk 1]
```diff
@@ -413,6 +413,16 @@ json-file | f
         else
             is "$output" "" "LogPath (driver=$driver)"
         fi
+
+	run_podman ? logs myctr
```
Comment (rhatdan) [inline test/system/030-run.bats:417]: Tab issue?

**Review by rhatdan — APPROVED** [review]

File: test/system/030-run.bats (hunk 1 above)
Comment (ypu) [reply test/system/030-run.bats]: Thanks for reminder. Updated.

File: test/system/030-run.bats [hunk 2]
```diff
@@ -413,6 +413,16 @@ json-file | f
         else
             is "$output" "" "LogPath (driver=$driver)"
         fi
+
+        run_podman ? logs myctr
+        if [[ $driver != 'none' ]]; then
+            is "$output" "$msg" "check that podman logs works as expect"
```
Comment (TomSweeneyRedHat) [inline test/system/030-run.bats:419]: nit, only if you need to make other changes:
```suggestion
            is "$output" "$msg" "check that podman logs works as expected"
```

File: test/system/030-run.bats [hunk 3]
```diff
@@ -413,6 +413,16 @@ json-file | f
...
+            [[ $status == 0 ]]
```
Comment (edsantiago) [inline test/system/030-run.bats:420]: The bats `[[` usage is evil. Please do this instead:
```bash
if [[ $driver == 'none' ]]; then
    run_podman 125 logs myctr
   ...
else
    run_podman logs myctr
   ...
fi
```

**Comment by ypu** [comment]
Thanks @TomSweeneyRedHat and @edsantiago for your comments. Update the code, also fixed the check failed for --log-driver none in remote test.

**Comment by edsantiago** [comment]
/lgtm

Thank you!

**Pull request merged**